frontend/src/libs/components/scroll-to-top-button/scroll-to-top-button.tsx

```diff
@@ -0,0 +1,58 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useEffect, useState } from '~/libs/hooks/hooks.js';
+
+import { IconButton } from '../icon-button/icon-button.js';
+import { SCROLL_VALUE } from './libs/enums.js';
+import styles from './styles.module.scss';
+
+const ScrollToTop: React.FC = () => {
+  const [hasScrollToTopButton, setHasScrollToTopButton] = useState(false);
+  const [scrollValue, setScrollValue] = useState(SCROLL_VALUE.TOP as number);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      if (window.scrollY > SCROLL_VALUE.SCROLLED_BELOW_200) {
+        setHasScrollToTopButton(true);
+
+        const position = window.scrollY;
+        const height =
+          document.documentElement.scrollHeight -
+          document.documentElement.clientHeight;
+
+        setScrollValue(Math.round((position / height) * 100));
```

Comment:
Could you, please, move this calculation to a separate variable to prevent magic numbers confusion?